frontend/src/modules/machinery/components/Translation.js

```diff
@@ -78,11 +81,15 @@ export default class Translation extends React.Component<Props> {
                     </ul>
                 </header>
                 <p className="original">
-                    <TranslationPlaceablesDiff
-                        diffTarget={ translation.original }
-                    >
-                        { sourceString }
-                    </TranslationPlaceablesDiff>
+                    { types.indexOf('Caighdean') === -1 ?
+                        <TranslationPlaceablesDiff
+                            diffTarget={ translation.original }
+                        >
+                            { sourceString }
+                        </TranslationPlaceablesDiff>
+                    :
+                        sourceString
+                    }
```

Comment:
This fix is outside the scope of the bug, but it's easier to test when Caighdean actually works.

Reply:
I don't understand, why disable diff and placeables when the source is Caighdean? Can you add a comment explaining that?

Reply:
Added explanation.